watertap/examples/flowsheets/full_treatment_train/util.py

```diff
@@ -22,7 +22,7 @@
 
 def solve_block(blk, solver=None, tee=False, fail_flag=True):
     if solver is None:
-        solver = get_solver(options={'tol':1e-11})
+        solver = get_solver()
     results = solver.solve(blk, tee=tee)
```

Comment:
One of the chemistry post-treatment blocks is failing a solution check after the tolerance was changed. The model still reaches an optimal solution, but the value is very slightly off now. The failure in the solution check is probably a negligible difference between the values, but I will need to go into it deeper to be certain.

Reply:
Looking at the documentation for IPOPT options, I see the default for this convergence tolerance is 1e-8, so I suppose the 1e-11 tolerance yielded a solution with smaller NLP error. As long as we still get "optimal termination" and the solutions are negligibly different, I'd think we'd be better off sticking with the default value of 1e-8.

Reply:
I think we should find the value that will give us reliable results for the chemistry models. By default IDAES relaxes this tolerance to `1e-06`, because that makes the most sense for the typical power plant model. We can, and in my opinion, should make a similar choice.

I'll be working on this today to see if there's a set of global options that works across our WaterTAP models.

Reply:
I think there should be a way to overwrite the IDAES default scaling for WaterTap (i.e. set out own set of defaults). John Eslick should know how to do this.